### PR TITLE
Elp

### PR DESCRIPTION
**Pull request by andmatei**

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -35,11 +38,14 @@
...
           <item index="10" class="java.lang.String" itemvalue="com.android.annotations.NonNull" />
+          <item index="11" class="java.lang.String" itemvalue="org.eclipse.jdt.annotation.NonNull" />
+          <item index="12" class="java.lang.String" itemvalue="io.reactivex.annotations.NonNull" />
+          <item index="13" class="java.lang.String" itemvalue="io.reactivex.rxjava3.annotations.NonNull" />
         </list>
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (andmatei) [inline .idea/misc.xml:48]: Me no laiki

File: app/src/main/java/com/girlesc/enguard/contracts/SignUpContract.java [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package com.girlesc.enguard.contracts;
 
-import com.girlesc.enguard.BasePresenter;
-import com.girlesc.enguard.BaseView;
+import com.girlesc.enguard.common.BasePresenter;
```
Comment (andmatei) [inline app/src/main/java/com/girlesc/enguard/contracts/SignUpContract.java:3]: MMMMMMM

File: .idea/misc.xml (hunk 1 above)
Comment (andmatei) [reply .idea/misc.xml:48]: I don't agree

File: .idea/misc.xml [hunk 3]
```diff
@@ -35,11 +38,14 @@
...
           <item index="9" class="java.lang.String" itemvalue="org.checkerframework.checker.nullness.compatqual.NonNullType" />
           <item index="10" class="java.lang.String" itemvalue="com.android.annotations.NonNull" />
+          <item index="11" class="java.lang.String" itemvalue="org.eclipse.jdt.annotation.NonNull" />
+          <item index="12" class="java.lang.String" itemvalue="io.reactivex.annotations.NonNull" />
+          <item index="13" class="java.lang.String" itemvalue="io.reactivex.rxjava3.annotations.NonNull" />
         </list>
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (LumiNitescu) [inline .idea/misc.xml:42]: no

**Review by LumiNitescu — CHANGES_REQUESTED** [review]

File: .idea/misc.xml (hunk 3 above)
Comment (andmatei) [reply .idea/misc.xml:42]: No you

**Pull request merged**